docrepr/sphinxext/__init__.py

```diff
@@ -1 +1,11 @@
- 
+from warnings import warn
+
+warn(
+    """
+    docrepr.sphinxext.plot_directive is deprecated and will eventually be removed;
+    please use matplotlib.sphinxext.plot_directive instead
+    """,
```

Comment:
```suggestion
warn(
    "docrepr.sphinxext.plot_directive is deprecated and will eventually be "
    "removed; please use matplotlib.sphinxext.plot_directive instead",
```

As confirmed in my testing, the triple-quoted string literal here preserves line breaks and whitespace, thus messing with the formatting—sorry I didn't notice this before! This fixes it, which I've also tested.

Reply:
Thanks!